add_item.php

```diff
@@ -14,7 +23,7 @@
     </header>
 
     <div class="form_container">
-		<form method="post">
+		<form method="post" action="./add_item.php">
```

Comment:
The action attribute here is actually not needed. Because you're pointing the form back to this page, leaving out the action attribute entirely will have the same effect. The default if no action is provided is to use the current page.

Reply:
ok done thanks